src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -0,0 +1,176 @@
+
+    @FindBy(how = How.ID, using = "acc13")
+    private WebElement tickerSymbolTextField;
+
+    @FindBy(how = How.ID, using = "acc14")
+    private WebElement ownershipList;
+
+    @FindBy(how = How.ID, using = "acc15")
+    private WebElement employeesTextField;
+
+    @FindBy(how = How.ID, using = "acc16")
+    private WebElement sicCodeTextField;
+
+    @FindBy(how = How.CSS, using = ".pbButton > input[name='save']")
+    private WebElement saveNewAccountButton;
+
+    /**
+     * Constructor.
+     */
+    public SFANewModifyPage() {
+        super();
+        PageFactory.initElements(this.webDriver, this);
+    }
+
+    /**
+     * Getter of saveNewAccountButton.
+     * @return button.
+     */
+    public WebElement getSaveNewAccountButton() {
+        return CommonActions.getElement(this.saveNewAccountButton);
+    }
+
+    /**
+     * Method to set accountName attribute.
+     * @param accountName string value.
+     */
+    public void setAccountNameTextField(final String accountName) {
+        this.accountNameTextField.sendKeys(accountName);
+    }
+
+    /**
+     * Method to set parentAccount attribute.
+     * @param parentAccount the value.
+     */
+    public void setParentAccountTextField(final String parentAccount) {
+        this.parentAccountTextField.sendKeys(parentAccount);
+    }
+
+    /**
+     * Method to set accountNumber attribute.
+     * @param accountNumber the value.
+     */
+    public void setAccountNumberTextField(final String accountNumber) {
+        this.accountNumberTextField.sendKeys(accountNumber);
+    }
+
+    /**
+     * Method to set accountSite attribute.
+     * @param accountSite the value.
+     */
+    public void setAccountSiteTextField(final String accountSite) {
+        this.accountSiteTextField.sendKeys(accountSite);
+    }
+
+    /**
+     * Method to set type attribute.
+     * @param type the value.
+     */
+    public void setAccountTypeList(final String type) {
+        this.typeList.sendKeys(type);
+    }
+
+    /**
+     * Method to set industry attribute.
+     * @param industry the value.
+     */
+    public void setIndustryList(final String industry) {
+        this.industryList.sendKeys(industry);
+    }
+
+    /**
+     * Method to set annualRevenue attribute.
+     * @param annualRevenue the value.
+     */
+    public void setAnnualRevenueTextField(final String annualRevenue) {
+        this.annualRevenueTextField.sendKeys(annualRevenue);
+    }
+
+    /**
+     * Method to click button.
+     */
+    public void clickSaveNewAccountButton() {
+        getSaveNewAccountButton().click();
+    }
+
+    /**
+     * Method to fill map with field name and method to set it.
+     * @return the map.
+     */
+    public Map<String, Value> fillMethodsToFields() {
+        newAccountsFields.put("accountName", (String v) -> {
+            setAccountNameTextField(v);
+        });
+        newAccountsFields.put("parentAccount", (String v) -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)